src/main/java/frc/robot/utils/AbsoluteEncoderChecker.java

```diff
@@ -0,0 +1,46 @@
+package frc.robot.utils;
+
+import edu.wpi.first.math.filter.MedianFilter;
+
+/** Checks Absolute Encoders for Noise */
+public class AbsoluteEncoderChecker {
+  private double[] buffer;
+  private int index;
+  private MedianFilter medianFilter;
+  private double lastCalculatedMedian = 0.0;
+  private static final int NUM_READINGS = 20;
```

Comment:
Can we put a TODO to rethink this whole thing since it was a nice idea but didn't really work